### DISCUSSION TITLE
Is the Client return capped?

### DISCUSSION THREAD

**Discussion by fangmarks** [supabase/supabase#1167] [Questions] [answered]
I'm trying to get the amount of how many entries in my image table have a specific folder.

In the SQL Editor I get 6100 using this Query
```
SELECT COUNT(*) from images where folder = 'wolves' 
```
but when I use `supabase.from<Image>('images').select().filter('folder', 'eq', 'wolves')` and then `data.length` in my code, it doesn't get more than 1000.

So my question is, is the amount of entries you get capped?

**Comment by fangmarks** [comment]
managed to get the actual count by changing my select function to `select("*", {count:"exact"})`

**Reply by fangmarks** [reply]
Still a bit bad though, since I don't think I can get all the IDs of the rows though?

**Comment by steve-chavez** [comment, marked as answer]
Yes, there's a max of 1000 rows. You can change it on `/settings/api` -> MAX ROWS.

**Reply by fangmarks** [reply]
Thanks!
